plugins/language_json.lua

```diff
@@ -0,0 +1,28 @@
+-- mod-version:3 priority:90
+
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "JSON",
+  files = { "%.json$" },
+  comment = nil,
+  patterns = {
+
+    { pattern = '\"[a-zA-Z0-9_]*\":', type = "keyword" }, -- key (string)
+    { pattern = '[0-9_]*:', type = "keyword" }, -- key (int)
```

Comment:
These should probably allow for spaces between the key and `:`.

Also I think you can avoid escaping `"` here.

Are numeric keys allowed in JSON?

Reply:
You're right, numeric keys are not allowed according to the JSON spec. Got them confused with how python handles them.